optimize/backend/src/main/java/io/camunda/optimize/service/importing/event/handler/EventImportIndexHandlerRegistry.java

```diff
@@ -22,23 +20,13 @@ public class EventImportIndexHandlerRegistry {
 
   private final BeanFactory beanFactory;
 
-  private final Map<String, CamundaEventTraceImportIndexHandler> camundaEventTraceHandlers =
-      new HashMap<>();
-
   public List<EngineImportIndexHandler<?, ?>> getAllHandlers() {
     final List<EngineImportIndexHandler<?, ?>> result = new ArrayList<>();
     result.add(getExternalEventTraceImportIndexHandler());
-    result.addAll(camundaEventTraceHandlers.values());
     return result;
   }
```

Comment:
perhaps this interface could be simplified now we know there is a single handler

Reply:
Think this can be part of the Q1 follow up